keras_preprocessing/sequence.py

```diff
@@ -39,12 +39,13 @@ def pad_sequences(sequences, maxlen=None, dtype='int32',
         sequences: List of lists, where each element is a sequence.
         maxlen: Int, maximum length of all sequences.
         dtype: Type of the output sequences.
+            To pad sequences with variable length strings, you can use `object`.
         padding: String, 'pre' or 'post':
             pad either before or after each sequence.
         truncating: String, 'pre' or 'post':
             remove values from sequences larger than
             `maxlen`, either at the beginning or at the end of the sequences.
-        value: Float, padding value.
+        value: Float or String, padding value.
```

Comment:
We would need some error handling.
If the users set value='pad', but doesn't set `dtype`, they would get a cryptic error message.

Reply:
Thanks. Good point.
It can be tricky to check this and handle all of the cases of incompatible dtypes. Do you have any suggestions on a nice way to check for errors?

Reply:
I added a warning.